modules/manage/partials/monitor-connectors.adoc

```diff
@@ -41,6 +41,30 @@ The most important metrics to be monitored by alerts are:
 * connector failed tasks
 * connector lag / connector lag rate
 
+== View connector logs
+
+Connector logs are written to the system topic `__redpanda.connectors_logs`. In {ui}, you can view them or pull them with `rpk`. For example:
```

Comment:
In this context, what's the difference between using rpk to view the logs versus pull the logs? Does "pull" mean hooking this up to Grafana? If so, is the process similar to monitoring clusters as described here? https://deploy-preview-1350--redpanda-docs-preview.netlify.app/redpanda-cloud/manage/monitor-cloud